mne/viz/topomap.py

```diff
@@ -1134,19 +1135,34 @@ def plot_evoked_topomap(evoked, times=None, ch_type=None, layout=None,
 
     if isinstance(axes, plt.Axes):
         axes = [axes]
-    if times is None:
+
+    if times == "peaks":
+        from scipy.signal import argrelmax
+        gfp = evoked.data.std(axis=0)
+        order = int(evoked.info["sfreq"] / 5.)
+        peaks = argrelmax(gfp, order=order)
+        if len(peaks) > 10:
+            peaks = peaks[np.sort((-peaks).argsort()[:10])]
```

Comment:
Actually, thinking about it ... in case of more than 10 peaks, you probably rather want to decrease the order and run again. Cause if you go by GFP maxima, you'll not find local maxima, but absolute maxima. Thoughts?